cogs/currency.py

```diff
@@ -6,20 +6,29 @@
 class Currency(commands.Cog):
     def __init__(self, bot):
         self.bot = bot
-        self.c = sqlite3.connect('currency.db')
-        self.connect = self.c.cursor()
+        self.conn = sqlite3.connect('currency.db')
+        self.c = self.conn.cursor()
+
+    def modify_balance(self, author, target, amount):
+        self.c.execute(f"UPDATE CURRENCY SET balance = balance+{amount} where (UID={author})")
```

Comment:
f strings open us up for SQL injection. Give this a read.

https://www.reddit.com/r/learnpython/comments/9kjw47/f_string_and_sql/